cumulusci/cli/cci.py

```diff
@@ -1203,7 +1218,7 @@ def flow_run(config, flow_name, org, delete_org, debug, o, skip, no_prompt):
     try:
         coordinator = config.get_flow(flow_name, options=options)
         coordinator.run(org_config)
-    except CumulusCIUsageError as e:
+    except (CumulusCIUsageError, FlowNotFoundError) as e:
```

Comment:
FlowNotFoundError is a subclass of CumulusCIUsageError; you don't have to catch it explicitly.